docker-compose.yml

```diff
@@ -1,63 +1,131 @@
+      SSO_AUTHORIZATION_RESPONSE_TYPE: code
+      SSO_AUTHORIZATION_SCOPE: openid
+      SSO_TOKEN_GRANT_TYPE: authorization_code
+      KC_IDP_HINT: idir
+      ENABLE_GOLD: true
+      MAINTENANCE_MODE_ACTIVE: false
+      SSO_REQUESTS_BACKEND_HOSTNAME: sso-requests
+      APP_URL: http://localhost:3000
     depends_on:
       - sso-requests
-    network_mode: host
-    volumes:
-      - ./app:/app/app
-      - app-node-modules:/app/app/node_modules/
+    networks:
+      - css-net
 
   sso-requests:
     container_name: sso-requests
     image: sso-requests:latest
     entrypoint: ['/bin/bash', '-c', 'yarn --cwd ./localserver migrate-db && yarn --cwd ./localserver dev']
+    ports:
+      - 8080:8080
     build:
       context: .
       dockerfile: Dockerfile
     depends_on:
       - sso-db
-    env_file:
-      - ./localserver/.env
-    network_mode: host
-    volumes:
-      - ./lambda:/app/lambda
-      - /app/lambda/node_modules/
-      - ./lambda/shared:/app/lambda/shared
-      - /app/lambda/shared/node_modules/
-      - ./lambda/app:/app/lambda/app
-      - /app/lambda/app/node_modules/
-      - ./lambda/actions:/app/lambda/actions
-      - /app/lambda/actions/node_modules/
-      - ./lambda/db:/app/lambda/db
-      - /app/lambda/db/node_modules/
-      - ./lambda/scheduler:/app/lambda/scheduler
-      - /app/lambda/scheduler/node_modules/
-      - ./localserver:/app/localserver
-      - /app/localserver/node_modules/
-      - ./app:/app/app
-      - app-node-modules:/app/app/node_modules/
+    environment:
+      PORT: 8080
+      APP_URL: http://localhost:3000
+      APP_ENV: local
+      API_AUTH_SECRET: s3cr3t
+      NODE_ENV: development
+      ALLOW_SILVER: false
+      ALLOW_GOLD: true
+      DATABASE_URL: postgresql://postgres:postgres@sso-db:5432/ssorequests
+      SSO_CLIENT_ID: css-app-in-gold-4128
+      SSO_CONFIGURATION_ENDPOINT: https://dev.loginproxy.gov.bc.ca/auth/realms/standard/.well-known/openid-configuration
+      KEYCLOAK_V2_DEV_USERNAME: admin
+      KEYCLOAK_V2_DEV_PASSWORD: admin
+      KEYCLOAK_V2_DEV_URL: http://dev-keycloak:9080
+      GH_BRANCH: main
+      GH_OWNER: bcgov
+      CHES_API_ENDPOINT: https://ches.api.gov.bc.ca/api/v1/email
+      CHES_TOKEN_ENDPOINT: https://loginproxy.gov.bc.ca/auth/realms/comsvcauth/protocol/openid-connect/token
+      CHES_USERNAME: test
+      CHES_PASSWORD: test
+      VERIFY_USER_SECRET: s3cr3t
+      API_URL: http://localhost:8080
+      REALM_REGISTRY_API: https://realm-registry-sandbox.apps.gold.devops.gov.bc.ca/api
+      LOCAL_DEV: true
+      GRAFANA_API_TOKEN: s3cr3t
+      GRAFANA_API_URL: https://sso-grafana-sandbox.apps.gold.devops.gov.bc.ca/api
+      GOLD_IP_ADDRESS: '142.34.229.4'
+    networks:
+      css-net:
+        aliases: [sso-requests-api]
+
+  dev-keycloak:
+    container_name: dev-keycloak
+    image: ghcr.io/bcgov/sso:dev
+    # image: sso-keycloak:18.0.2
+    command: ['-Djboss.socket.binding.port-offset=1000']
```

Comment:
I was getting an error on startup with this being in the command. I moved it into the JAVA_OPTS_APPEND and it was working for me, `JAVA_OPTS_APPEND: '-Djboss.socket.binding.port-offset=1000 -Djboss.persistent.log.dir=/opt/eap'`

Reply:
Thanks for finding that @jlangy , I have updated it now.